include/ui/gui-project.hpp

```diff
@@ -231,6 +231,11 @@ class GuiProject : QObject {
    */
   void _throwError(const std::string& message,
                    const std::vector<std::string>& arguments);
+  /**
+   * Shows a runtime error in the ui
+   * \param message The translateable that contains the message
+   */
+  void _throwError(const Translateable& message);
```

Comment:
Deleting the old `_throwError` message and changing the calls would probably be good, but I can do this too, if you don't have time.